Application/EdFi.Ods.AdminApi/E2E Tests/Admin API E2E 2.0 - ClaimSets.postman_collection.json

```diff
@@ -1798,7 +1947,7 @@
 								"method": "GET",
 								"header": [],
 								"url": {
-									"raw": "{{API_URL}}/v2/claimSets/{{CreatedClaimSetId}}",
+									"raw": "{{API_URL}}/v2/claimsets/{{CreatedClaimSetId}}",
```

Comment:
@CSR2017
We changed the endpoints from "claimset" to "claimSet", but most of the services were changed again to "claimset" in this PR, Is that ok?